.jenkins

```diff
@@ -1,42 +1,19 @@
 #!/usr/bin/env groovy
 
-pipeline {
+// Include this shared CI repository to load script helpers and libraries.
+library identifier: 'vapor@1.0.0-RC10', retriever: modernSCM([
+  $class: 'GitSCMSource',
+  remote: 'https://github.com/vapor-ware/ci-shared.git',
+  credentialsId: 'vio-bot-gh',
+])
 
-    agent any
-
-    stages {
-        stage('Checkout') {
-            steps {
-                checkout scm
-            }
-        }
-
-        stage('Publish to PyPi') {
-            when {
-                buildingTag()
-            }
-            environment {
-                TWINE_USERNAME = 'vaporio'
-                TWINE_PASSWORD = credentials('twine-password')
-            }
-            steps {
-                sh 'tox -e publish'
-            }
-        }
-
-        stage('GitHub Release') {
-            when {
-                buildingTag()
-            }
-            environment {
-                GITHUB_USER = 'vapor-ware'
-                GITHUB_TOKEN = credentials('vio-bot-gh-token')
-                GITHUB_REPONAME = 'fake-rpigpio'
-            }
-            steps {
-                sh 'docker run --rm -v ${WORKSPACE}:/repo edaniszewski/ghr -u ${GITHUB_USER} -r ${GITHUB_REPONAME} -t ${GITHUB_TOKEN} -replace ${TAG_NAME} dist/'
-            }
-        }
-
-    }
-}
\ No newline at end of file
+pythonPipeline([
+  'pythonVersion': '3.8',
+  'twineCredential': 'pypi-token-fake-rpigpioo-upload',
```

Comment:
just making sure this isn't a typo.

Reply:
looks like it is.

`pypi-token-fake-rpigpio-upload`

Reply:
nice catch  -- thanks!

Reply:
This should now be fixed